### DISCUSSION TITLE
I get 404 error when trying access "Getting Started" colab

### DISCUSSION THREAD

**Discussion by somidi-ea** [google/meridian#174] [Q&A] [answered]
Getting 404 Not found for: 

https://developers.google.com/meridian/notebook/meridian-getting-started

**Comment by vivekbeniwal** [comment]
Hi @somidi-ea 

Can you reach out to meridian-comms@google.com with subject: "[Access Issue]" and someone will help you out with this.

Regards
Vivek

**Comment by JonasSchroederOtto** [comment, marked as answer]
I had the same issue. What resolved it for me was to open Google Colab separately and open a notebook connected to my GitHub through there. I was able to open the Getting Started notebook in Colab through there

**Reply by somidi-ea** [reply]
Thanks that solved the issue for me
